### ISSUE TITLE
Cable properties missing

### ISSUE THREAD

**Issue by jmcbxl** [samilliken/openDCIM#962]
Data about cabling is currently stored by using fields in "fac_ports table". Doing so requires to store some informations twice in the port table : "connected port", "connected device", "notes" need to be stored at each end of the cable.
Should be better to implement a "fac_cable" table and refering the cable by it's internal id.

This would allow some more possibilities for managing cabling : 
- fields for length, labels, electrical properties, connector type, ...
- generating unique labeling much more easily
- keep track of a cable that is not connected at one or both sides (either in the rack or in a box for future use, ...)

**Comment by miaxon** [comment]
very good idea.

**Comment by 0smansen** [comment]
I totally agree. Label property will fill the gap which I suffer at cabling. And also this field should be placed in "**View cable path**" diagram and reports like "**Network Map Viewer**".  Using notes in connection is not a useful workaround.
